app/controllers/course/admin/admin_controller.rb

```diff
@@ -11,6 +11,15 @@ def update #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
```

Comment:
Changing `current_course` to `@course` reduces the assignment factor by 4.

On Tuesday, February 9, 2016, Joel Low <notifications@github.com> wrote:

> In app/controllers/course/admin/admin_controller.rb
> <https://github.com/Coursemology/coursemology2/pull/765#discussion_r52280562>
> :
>
> > @@ -11,6 +11,15 @@ def update #:nodoc:
> >      end
> >    end
> >
> > +  def destroy #:nodoc:
>
> How does it solve it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/765/files#r52280562>.
>


-- 

Warmest Regards,
Weiqing Toh.
9297 3395